business-central-parent/business-central-webapp/pom.xml

```diff
@@ -2708,6 +2708,9 @@
             <compileSourcesArtifact>org.kie.workbench.stunner:kie-wb-common-stunner-forms-client</compileSourcesArtifact>
             <compileSourcesArtifact>org.kie.workbench.stunner:kie-wb-common-stunner-project-api</compileSourcesArtifact>
             <compileSourcesArtifact>org.kie.workbench.stunner:kie-wb-common-stunner-project-client</compileSourcesArtifact>
+	    <compileSourcesArtifact>org.kie.workbench.stunner:kie-wb-common-stunner-kogito-api</compileSourcesArtifact>
```

Comment:
You couldn't fix the formatting could you!? Sorry.

Reply:
why this needs to be added in BC? Is that API used in 7.x applications?
This is big -1 to include it in full and productized part of the buiild

Reply:
done @manstis